documentation/docs/02-routing.md

```diff
@@ -302,7 +302,9 @@ Files and directories with a leading `_` or `.` (other than [`.well-known`](http
 
 #### Rest parameters
 
-A route can have multiple dynamic parameters, for example `src/routes/[category]/[item].svelte` or even `src/routes/[category]-[item].svelte`. (Parameters are 'non-greedy'; in an ambiguous case like `/x-y-z`, `category` would be `x` and `item` would be `y-z`.) If the number of route segments is unknown, you can use rest syntax — for example you might implement GitHub's file viewer like so...
+A route can have multiple dynamic parameters, for example `src/routes/[category]/[item].svelte` or even `src/routes/[category]-[item].svelte`. (Parameters are 'non-greedy'; in an ambiguous case like `/x-y-z`, `category` would be `x` and `item` would be `y-z`.)
+
+But if the number of route segments is unknown, you can use rest syntax — for example you might implement GitHub's file viewer like so...
```

Comment:
it sounds a bit strange to start a paragraph with "But" to me as it seems like a new paragraph should be a new thought. I wonder if we even need the preceding paragraph at all?
```suggestion
If the number of route segments is unknown, you can use rest syntax. For example, you might implement GitHub's file viewer like so...
```

Reply:
turns out this is actually duplicative with text above. fixed: c84e11ff2877f2bcd4d43fb1ebdf24047ed358fa